### ISSUE TITLE
batch normalizationを試す

### ISSUE THREAD

**Issue by ogura-edu** [ogura-edu/HDNNP#79]
https://qiita.com/t-tkd3a/items/14950dbf55f7a3095600

**Comment by ogura-edu** [comment]
訓練時はミニバッチの平均と分散を使って正規化し、  
評価時には訓練時に計算しておいた平均と分散の「指数移動平均」を使う

**Comment by ogura-edu** [comment]
https://kratzert.github.io/2016/02/12/understanding-the-gradient-flow-through-the-batch-normalization-layer.html
